dotfiles/bash/aliases

```diff
@@ -383,20 +383,20 @@ function heroku_backup! {
 function heroku_capture {
   heroku_repo=`heroku_repo $1`
   heroku_backup_line=`heroku pg:backups capture --app $heroku_repo`
-  heroku_backup_id=`echo $heroku_backup_line | cut -d\  -f32`
+  heroku_timestamp_line=`heroku pg:backups --app $heroku_repo`
+  heroku_backup_id=`echo $heroku_timestamp_line | cut -d\  -f1 | head -4 | tail -1`
```

Comment:
ok thank you @claudiofullscreen , you are right this PR worked for about 3 days for me then stopped...I'll try the new one!